src/main/java/org/apache/maven/plugins/pmd/PmdReport.java

```diff
@@ -63,16 +63,19 @@
 @Mojo(name = "pmd", threadSafe = true, requiresDependencyResolution = ResolutionScope.TEST)
 public class PmdReport extends AbstractPmdReport {
     /**
-     * The target JDK to analyze based on. Should match the source used in the compiler plugin. Valid values
-     * with the default PMD version are
+     * The target JDK to analyze based on. Should match the source used in the compiler plugin.
+     * Valid values depend on the used PMD version. With the default PMD version valid values are
      * currently <code>1.3</code>, <code>1.4</code>, <code>1.5</code>, <code>1.6</code>, <code>1.7</code>,
      * <code>1.8</code>, <code>9</code>, <code>10</code>, <code>11</code>, <code>12</code>, <code>13</code>,
      * <code>14</code>, <code>15</code>, <code>16</code>, <code>17</code>, <code>18</code>, <code>19</code>,
-     * and <code>20</code>.
+     * <code>20</code>, <code>21</code>, and <code>22</code>.
```

Comment:
Are those old version before 7 still supported?

Reply:
Yes, they are still supported. But true, it's questionable, how useful these are. We could thin out the list and keep only 8, 11, 17, 21 and refer to https://docs.pmd-code.org/latest/pmd_languages_java.html for the full list.

Reply:
Let's do so.

Reply:
Done.